Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: build lint start
+
+build:
+	pipenv install --dev
+
+lint:
+	pipenv run flake8 ./ras_party ./test
+	pipenv check ./ras_party ./test
+
+start:
+	pipenv run python3 run.py
```

Comment:
Should we have a command for running tests as well?

Reply:
Added a test command